### PR TITLE
Add active links to Header component

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/N0tUjqnr/265-2-set-active-styles-on-header-nav-links

Allows the use of the `active` param to set which link, if any, should be active.

```
{{ dmHeader({
  "role": "supplier",
  "active": "/help"
})}}
```

Screenshot 2020-11-03 at 16 40 11

### CODE REVIEW

**Review by lfdebrux — APPROVED** [review]
Nifty! Could you add a note to the changelog explaining how to use it in Flask?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ See below for Changelog examples.
 
 ## 2.5.0
 
+🔧 Fixes:
```
Comment (lfdebrux) [inline CHANGELOG.md:7]: I'd call this a new feature

```suggestion
🆕 New features:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ See below for Changelog examples.
 
 ## 2.5.0
```
Comment (lfdebrux) [inline CHANGELOG.md:5]:
```suggestion
## Unreleased
```

**Comment by domoscargin** [comment]
> Nifty! Could you add a note to the changelog explaining how to use it in Flask?

Hmm, interesting - I figured this would just be set in the template.

```
{{ dmHeader({
  "role": "supplier",
  "active": {{ request.path }}
})}}
```

What were you looking for in the changelog?

**Comment by lfdebrux** [comment]
> > Nifty! Could you add a note to the changelog explaining how to use it in Flask?
> 
> Hmm, interesting - I figured this would just be set in the template.
> 
> ```
> {{ dmHeader({
>   "role": "supplier",
>   "active": {{ request.path }}
> })}}
> ```
> 
> What were you looking for in the changelog?

Exactly what you just wrote :D it's just nice to show usage examples, especially when it's using a neat little trick like that.

(Side note, Jinja doesn't let you have `{{` inside `{{`, it should just be `"active": request.path`.

**Comment by domoscargin** [comment]
Of course updating the CHANGELOG has made everything fail...

**Comment by lfdebrux** [comment]
> Of course updating the CHANGELOG has made everything fail...

:'(

This is #208 I'm guessing? We should probably tell standard to ignore uses of `var` for now, or at least make the tests not fail on warnings...

**Comment by domoscargin** [comment]
> > Of course updating the CHANGELOG has made everything fail...
> 
> :'(
> 
> This is #208 I'm guessing? We should probably tell standard to ignore uses of `var` for now, or at least make the tests not fail on warnings...

I don't think it is, actually - the rule is just a warning for now. This is failing because of a slow test which hasn't been too slow till now...

**Pull request merged**